test/asm/local-wrong-parent.err

```diff
@@ -1,3 +0,0 @@
-error: local-wrong-parent.asm(5):
```

Comment:
This whole test should be renamed or rewritten, since there's no longer a "wrong" parent".

Reply:
Actually yeah, this can be deleted and let sym-scope.asm cover everything.